### DISCUSSION TITLE
Product feedback: Title wrapping in pinned Discussions

### DISCUSSION THREAD

**Discussion by gabbybilka** [community/community#33743] [Discussions]
We've recently migrated to using the Discussions tab to post announcements on our repository (microsoft-ui-xaml) due to the improved threading format over issues. However, it's common that the titles of the announcements we post are too long to be visible when pinned, requiring the user to click into the discussion to understand the context. This is especially common when we keep 2+ discussions pinned. Issues when pinned have title wrapping enabled.

Why do the titles in pinned Discussions fail to wrap like the titles in pinned Issues?

**Comment by siminapasat** [comment]
Thanks for your feedback @gabbybilka ! We'll take it to the team and look into it 😊 

Just to make sure -- are you referring to this view (the pinned announcement at the top)?
<img width="1118" alt="image" src="https://user-images.githubusercontent.com/16675781/191673469-172657e9-d7cf-4af0-9935-3a15e657ed8a.png">

**Reply by gabbybilka** [reply]
Yep, this view. I unpinned the issue I had up before to emphasize the one shown above:
![Screenshot 2022-09-22 091229](https://user-images.githubusercontent.com/20007011/191798601-0ddc5896-ddfe-4dd8-ab72-59ab01b1af4e.png)

**Reply by siminapasat** [reply]
ACK! I noticed it on our own discussions as well, very easy to repro 😉
